locales/zh_CN/LC_MESSAGES/contributor_guide.po

```diff
@@ -155,825 +164,801 @@ msgstr "单元/集成测试是我们的好伙伴。"
 
 #: ../../contributor_guide.rst:41
 msgid ""
-"“There are two ways of constructing a software design: One way is to make"
-" it so simple that there are obviously no deficiencies, and the other way"
-" is to make it so complicated that there are no obvious deficiencies. "
-"`The first method is far more difficult`.”     --- `C.A.R. Hoare "
-"<https://en.wikipedia.org/wiki/Tony_Hoare>`_"
+"“There are two ways of constructing a software design: One way is to make it so "
+"simple that there are obviously no deficiencies, and the other way is to make "
+"it so complicated that there are no obvious deficiencies. `The first method is "
+"far more difficult`.”     --- `C.A.R. Hoare <https://en.wikipedia.org/wiki/"
+"Tony_Hoare>`_"
 msgstr ""
+"“软件设计过程中中存在两种模式：一种是使之结构简单明了到没有任何问题，另一种是令"
+"结构设计足够复杂到完美无缺 `而第一种方案则要困难的多`.”     — `C.A.R. 霍尔 "
+"<https://en.wikipedia.org/wiki/Tony_Hoare>`_"
 
 #: ../../contributor_guide.rst:44
 msgid "Effective communication"
 msgstr "高效率地沟通"
 
 #: ../../contributor_guide.rst:46
 msgid ""
-"How much information we effectively convey, is way more important than "
-"how many words we typed."
-msgstr "传达了多少有效信息，比我们打了多少字更重要。"
+"How much information we effectively convey, is way more important than how many "
+"words we typed."
+msgstr "传达了多少有效信息，比我们打了多少字更重要的多。"
 
 #: ../../contributor_guide.rst:47
 msgid "Be constructive. Be polite. Be organized. Be concise."
-msgstr "积极，有礼貌，有组织性，准确性。"
+msgstr "在沟通中保持积极，礼貌，注意语言的组织性、准确性。"
 
 #: ../../contributor_guide.rst:48
 msgid "Bulleted lists are our friends."
-msgstr "无序列表/列清单是我们的好伙伴。"
+msgstr "注意除了文字之外，列表(Bulleted lists)也是我们表达过程中的好伙伴。"
 
 #: ../../contributor_guide.rst:49
 msgid ""
-"Proofread before you post: if you are the reader, can you understand what"
-" you typed?"
+"Proofread before you post: if you are the reader, can you understand what you "
+"typed?"
 msgstr "提交评论前请仔细预读：如果你是读者，你能读懂自己所写的内容么？"
 
 #: ../../contributor_guide.rst:50
 msgid ""
 "If you are not a native speaker, consider using a spell checker such as "
 "`Grammarly <https://app.grammarly.com/>`_."
-msgstr "如果你的母语不是英语，考虑使用拼写检查器，如 `Grammarly <https://app.grammarly.com/>`_ 。"
+msgstr ""
+"如果你的母语不是英语，考虑使用拼写检查器，如 `Grammarly <https://app.grammarly."
+"com/>`_ 。"
 
 #: ../../contributor_guide.rst:54
 msgid "Making good pull requests"
-msgstr "做出良好的 PR"
+msgstr "提交良好的 PR"
 
 #: ../../contributor_guide.rst:56
 msgid ""
-"PRs with **small** changesets are preferred. A PR should ideally address "
-"**only one issue**."
-msgstr "我们鼓励变化很小的 PR， 一个 PR 理想情况下应该 **只针对一个问题/issue** 。"
+"PRs with **small** changesets are preferred. A PR should ideally address **only "
+"one issue**."
+msgstr ""
+"我们鼓励改动很小的 PR， 一个 PR 理想情况下应该 **只针对一个问题(issue)** 。"
 
 #: ../../contributor_guide.rst:58
 msgid ""
-"It is fine to include off-topic **trivial** refactoring such as typo "
-"fixes;"
-msgstr ""
+"It is fine to include off-topic **trivial** refactoring such as typo fixes;"
+msgstr "对于一些 **似乎不是很重要** 的优化重构，比如排版上的建议，也是很受欢迎的;"
```

Comment:
```suggestion
msgstr "也可以参杂一些 **无关紧要** 的优化重构，比如修正笔误;"
```
When verifying translation, we check **mean-by-mean** to prevent infomation loss. For example:
`fine` -> `受欢迎` (good)
`off-topic trivial` -> `似乎不是很重要` (good)
`typo` -> `排版` (wrong translation)
`include` -> `???` (not translated at all)
And you see, the infomation of `include` is lost during translation.

The meaning of this entry is, for example, you fix an typo in `README.md` in this PR, although the PR topic is to translate `contributor_guide`.

Self-reviewing&polishing the translation with this method before the reviewer take part in, could siginificant reduce the reviewer's stress, and put his/her energy&time to those really hard and discussion-needed issues.

Reply:
I'm sorry for the increased workload.
In fact, my workflow before I submit includes:

- A word-for-word translation

- Check layout, symbols

- Double check to enhance expression

- Finally review the document again

- Commit a PR
The workload of this document is a little higher than before.Then I may not stay focused in the later stages. I apologize for any inconvenience I have caused.🙏 I will pay attention to correct according to your suggestion in the following work.